hbase/test_hbase.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from parameterized import parameterized
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class HBaseTestCase(DataprocTestCase):
+    COMPONENT = 'hbase'
+    INIT_ACTION = 'gs://dataproc-initialization-actions/hbase/hbase.sh'
+
+    def verify_instance(self, name):
+        ret_code, stdout, stderr = self.run_command(
+            'gcloud compute ssh {} --command '
+            '"/etc/hbase/bin/hbase org.apache.hadoop.hbase.IntegrationTestsDriver -r {}"'.format(
+                name,
+                "org.apache.hadoop.hbase.mapreduce.IntegrationTestImportTsv"
+            )
+        )
+        self.assertEqual(ret_code, 0, "Failed to validate cluster. Error: {}".format(stderr))
+
+    """
+    Single and standard mode on 1.0 and 1.1 are disabled because HBase doesn't work in those 
+    modes on older dataproc versions. HBase is intended to run on HA environment.
+    HA mode on 1.0 and 1.1 doesn't pass integration test.
+    """
+    @parameterized.expand([
+        # ("SINGLE", "1.0", ["m"]),
+        # ("STANDARD", "1.0", ["m"]),
```

Comment:
Let's remove these comments.

Reply:
Thanks @aniket486 for a review. Not sure if I understood correctly, but I removed commented lines, let me know if it's ok now.